client/extensions/woocommerce/app/shipping/shipping-card.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n from 'i18n-calypso';
```

Comment:
Fwiw, a lot of components are using this technique, especially for simpler components: https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/store-sidebar/store-ground-control.js#L46